redfish-core/lib/account_service.hpp

```diff
@@ -2001,6 +2026,28 @@ inline void requestAccountServiceRoutes(App& app)
                 }
             }
 
+
+            // Don't PATCH any account which has a Restricted Role,
+            // except allow the Locked property.
```

Comment:
What is the difference between Locked and Enabled? 
Why did we chose Locked over Enabled ?

Reply:
See Redfish [ManagerAccount](https://redfish.dmtf.org/schemas/v1/ManagerAccount.v1_8_1.json) for definitions of the Locked and Enabled properties.
- Locked - The service account participates in the account lockout policy (too many bad passwords entered) and we need to allow the admin to unlock the service account.
- Enabled - We are not giving the administrator the ability to enable or disable the "service" account.  Instead, the admin controls when the ACF file is uploaded and deleted.

Reply:
I am confused because here you said 
"The BMC admin is allowed to enable/disable the “service” account, but IBM recommendations will be to leave the “service” account **enabled**, and only paranoid users will want to disable it."

https://ibm-systems-power.slack.com/archives/C0Q6TQP5Z/p1641830896072100?thread_ts=1641827883.070300&cid=C0Q6TQP5Z

Reply:
I'll change it to allow both Locked and Enabled to be changed for users who have a Restricted role.
Note that our thinking changed several times on the question if we should allow the Enabled property to be changed.  (Sorry about that.) The current design allows the BMC admin to disable and re-enable such users.